interface/billing/billing_report.php

```diff
@@ -411,927 +411,949 @@ function MarkAsCleared(Type) {
     document.onclick = TakeActionOnHide;
 </script>
 <!-- =============Included for Insurance ajax criteria==== -->
+<?php
+
+//to determine and set the form to open in the desired state - expanded or centered, any selection the user makes will become the user-specific default for that page
+// collectAndOrganizeExpandSetting() contains a single array as an argument, containing one or more elements, the name of the current file is the first element, if there are linked 
+// files they should be listed thereafter, please add _xpd suffix to the file name
+
```

Comment:
Lets clean up the comment a bit (just so the width of the text is standard) and also remove the empty line between the comment and the 3 lines of code
(yep, i know, being way too much of a perfectionist :) )